package-dev/Plugins/macOS/SentryNativeBridge.m

```diff
@@ -46,6 +46,8 @@ int SentryNativeBridgeLoadLibrary()
 {
     NSMutableDictionary *dictOptions = [[NSMutableDictionary alloc] init];
     dictOptions[@"sdk"] = @ { @"name" : @"sentry.cocoa.unity" };
+    dictOptions[@"enableAutoSessionTracking"] = @NO;
```

Comment:
I've noticed this could be here statically, instead of setting it with a native interop call